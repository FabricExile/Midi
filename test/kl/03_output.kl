require Midi;

operator entry() {
  MidiOutput midi();
  report(midi.getPortCount());
  report(midi.isPortOpen());
  midi.openPort();
  report(midi.isPortOpen());
  Byte message[];
  message.resize(3);
  for(Size i=0;i<128;i++) {
    message[0] = (MIDI_CONTROL_CHANGE);
    message[1] = (Byte(i));
    message[2] = (0);
    midi.sendMessage(message);
  }
  report('completed!'); 
}