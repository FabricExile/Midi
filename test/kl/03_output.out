[FABRIC:MT] 2
[FABRIC:MT] false
[FABRIC:MT] true
[FABRIC:MT] completed!
